flush_interval raises an exception

If you configure flush_interval: 0, then the beat is crashing.

The configuration file looks like:
output:

  # Elasticsearch as output
  # Options:
  # host, port: where Elasticsearch is listening on
  # save_topology: specify if the topology is saved in Elasticsearch
  elasticsearch:
    enabled: true
    hosts: ["http://localhost:9200"]
    flush_interval: 0

panic: non-positive interval for NewTicker

goroutine 1 [running]:
time.NewTicker(0x0, 0x3e8)
	/usr/local/Cellar/go/1.5.1/libexec/src/time/tick.go:23 +0xe4
github.com/elastic/libbeat/publisher.newBulkWorker(0xc82128e100, 0x3e8, 0x60b9710, 0xc82128c140, 0x0, 0x2710, 0x9)
	/Users/monica/src/github.com/elastic/packetbeat/Godeps/_workspace/src/github.com/elastic/libbeat/publisher/bulk.go:31 +0x4c
github.com/elastic/libbeat/publisher.asyncOutputer(0xc82128e100, 0xc82128c140, 0x0, 0x0)
	/Users/monica/src/github.com/elastic/packetbeat/Godeps/_workspace/src/github.com/elastic/libbeat/publisher/async.go:96 +0x265
github.com/elastic/libbeat/publisher.newAsyncPublisher(0x489a5e0, 0xc8212129c0)
	/Users/monica/src/github.com/elastic/packetbeat/Godeps/_workspace/src/github.com/elastic/libbeat/publisher/async.go:32 +0x15a
github.com/elastic/libbeat/publisher.(*PublisherType).Init(0x489a5e0, 0x45c6cd0, 0xa, 0xc82125b620, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/Users/monica/src/github.com/elastic/packetbeat/Godeps/_workspace/src/github.com/elastic/libbeat/publisher/publish.go:266 +0xd92
github.com/elastic/libbeat/beat.(*Beat).LoadConfig(0xc820014460)
	/Users/monica/src/github.com/elastic/packetbeat/Godeps/_workspace/src/github.com/elastic/libbeat/beat/beat.go:96 +0x34f
main.main()
	/Users/monica/src/github.com/elastic/packetbeat/main.go:26 +0x27e

--- a/publisher/async.go
+++ b/publisher/async.go
@@ -87,7 +87,7 @@ func asyncOutputer(ws *workerSignal, worker *outputWorker) worker {
 	}
 
 	// batching disabled
-	if flushInterval < 0 || maxBulkSize < 0 {
+	if flushInterval <= 0 || maxBulkSize <= 0 {
 		return worker
 	}
 

